### PR TITLE
Feature/price confirmator

### PR DESCRIPTION
**Pull request by screamcha**

### CODE REVIEW

File: server/migrations/sqls/20180307060520-startup-up.sql [hunk 1]
```diff
@@ -392,13 +392,10 @@ create or replace function create_order(fid integer, uid integer, exp timestamp)
...
 	into onum
 	from orders
-	where user_id = uid
-	order by order_number
-		desc
-	limit 1;
+	where user_id = uid;
 	
 	insert into orders(user_id, status, expires_at, order_number)
 		values(uid, 'Pending', exp, onum+1)
```
Comment (AndreiShenets) [inline server/migrations/sqls/20180307060520-startup-up.sql]: values shouldn't be indented

File: server/migrations/sqls/20180307060520-startup-up.sql [hunk 2]
```diff
@@ -451,4 +448,34 @@ begin
 	
 	return ret;
 end;
+$$ language plpgsql;
+
+create or replace function confirm_order(oid integer)
+	returns void as $$
+begin
+	update orders
+		set status = 'Confirmed'
```
Comment (AndreiShenets) [inline server/migrations/sqls/20180307060520-startup-up.sql]: set shouldn't be indented

File: server/src/Routes/Handlers/newFlight/placePicker.js [hunk 3]
```diff
@@ -46,6 +46,12 @@ const bookTemporarily = async ctx => {
       return;
     }
   }
+  if (!req.placeIds.length) {
```
Comment (AndreiShenets) [inline server/src/Routes/Handlers/newFlight/placePicker.js]: potential null ref here?

Comment (screamcha) [reply server/src/Routes/Handlers/newFlight/placePicker.js]: @AndreiShenets this issue is checked is a few lines above. Client'll receive 400 with another message in that case.

Comment (AndreiShenets) [reply server/src/Routes/Handlers/newFlight/placePicker.js]: I can explicitly send you null, cannot I? In this case when req[key] returns null the check req[key] === undefined returns false

**Pull request merged**